mobile/src/reducers/user/user-permissions.model.ts

```diff
@@ -0,0 +1,21 @@
+import { required, dataMember, typedArray } from 'santee-dcts';
+
+export class UserPermissions {
```

Comment:
please rename it to UserEmployeePermissions (you can also add a 'employeeId' to it, for debugging purposes). It's confusing now - feels like some kind of global 'approve' permission